nengo_gui/examples/tutorial/24-spa-unbinding.py

```diff
@@ -0,0 +1,23 @@
+# Tutorial 24: Unbinding Concepts
+
+import nengo
+import nengo.spa as spa
+
+D = 32  # the dimensionality of the vectors
+
+model = spa.SPA()
+with model:
+    model.color = spa.Buffer(D)
+    model.shape = spa.Buffer(D)
+    model.memory = spa.Memory(D, synapse=0.1)
+    model.query = spa.Buffer(D)
+    model.answer = spa.Buffer(D)
+
+    actions = spa.Actions(
```

Comment:
Good idea...  in fact, better yet, they can be removed.  They were just there to slow down how quickly things were shoved in to memory, but an even better way of doing that is increasing the synapse time a bit (since that improves the memory as well).